python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -145,7 +145,10 @@ def _common_submit(self, dirac_script):
         self.statusInfo = ''
         j.been_queued = False
         dirac_cmd = """execfile(\'%s\')""" % dirac_script
-        result = execute(dirac_cmd)
+
+        try:
+            result = execute(dirac_cmd)
+        except GangaDiracException as err:
```

Comment:
I've no idea the origin of these and tbh I'm tempted to remove it entirely as it's not used atm and hasn't been for a _long_ time.